lib/string.go

```diff
@@ -0,0 +1,23 @@
+package lib
+
+import (
+	"strings"
+)
+
+// StrContains checks if a list contains a string
+func StrContains(l []string, s string) bool {
+	for _, v := range l {
+		if v == s {
+			return true
+		}
+	}
+	return false
+}
+
+func ToLowerList(l []string) []string {
```

Comment:
That's what I thought, but it's used across several unit tests.  I didn't think it was necessary to create a `test-lib` package, but could if we wanted to be pedantic about the scope and use of this method.  I have no preference one way or another.